<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogoDatosGenerales</class>
 <widget class="QDialog" name="DialogoDatosGenerales">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>445</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>790</width>
    <height>445</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>790</width>
    <height>445</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>790</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Tab_DatosGenerales">
    <attribute name="title">
     <string>Datos Generales</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>781</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="comboListaCuadroPrecios">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelCuadroDePrecios">
          <property name="text">
           <string>Cuadro de precios (Codificación)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="comboBox_2">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Español</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Inglés</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Alemán</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Húngaro</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Eslovaco</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelIdiomas">
            <property name="text">
             <string>Idioma</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QComboBox" name="comboListaDatos"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="TablaDatosGenerales"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Tab_Coeficientes">
    <attribute name="title">
     <string>Coeficientes</string>
    </attribute>
    <widget class="QTableWidget" name="TablaCoeficientes">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>1</y>
       <width>441</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>320</y>
       <width>193</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="labelCostesIndirectos">
        <property name="text">
         <string>%Costes Indirectos</string>
        </property>
        <property name="buddy">
         <cstring>lineEditCostesIndirectos</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditCostesIndirectos">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>410</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogoDatosGenerales</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>591</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
